db/schema.rb

```diff
@@ -36,4 +44,11 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
+  create_table "value_sets", force: :cascade do |t|
```

Comment:
This is out of date. You should drop your db and run the migrations to get an accurate schema.

Reply:
resolved